lib/util/cache/repository/impl/local.ts

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs-extra';
+import upath from 'upath';
+import { GlobalConfig } from '../../../../config/global';
+import { logger } from '../../../../logger';
+import {
+  CACHE_REVISION,
+  canBeMigratedToV11,
+  isValidCacheRecord,
+} from '../common';
+import type { RepoCache, RepoCacheData, RepoCacheRecord } from '../types';
+
+export class LocalRepoCache implements RepoCache {
+  private data: RepoCacheData = {};
```

Comment:
can we use js private fields?
```suggestion
  private #data: RepoCacheData = {};
```

Reply:
Seems like it can be either `private` or `#`-private (for current class only)